kahuna/public/js/components/gr-delete-image/gr-delete-image.js

```diff
@@ -0,0 +1,33 @@
+import angular from 'angular';
+import '../gr-confirm-delete/gr-confirm-delete';
+import './gr-delete-image.css!';
+
+export const imageDelete = angular.module('gr.imageDelete', ['gr.confirmDelete']);
+
+imageDelete.controller('grImageDeleteCtrl', ['$scope', 'mediaApi', function ($scope, mediaApi) {
+    var ctrl = this;
+
+    ctrl.delete = function () {
+        mediaApi.delete(ctrl.image)
+            .then((resp) => ctrl.onSuccess(resp))
+            .catch((err) => ctrl.onError(err));
```

Comment:
Would this not error from instantiating the directive without them as you've required them in the [scope](https://github.com/guardian/grid/pull/1051/files#diff-98f40c89c6f7ce3607c23deb9b0b1856R29)?